module/components/Avatar/index.js

```diff
@@ -12,48 +12,64 @@ const propTypes = {
      */
     color: PropTypes.oneOf(['primary', 'secondary', 'success', 'info', 'warning', 'danger', 'link']),
     /**
-     * The icon of the component. It's using the theme palette when that makes sense.
+     * The labelText of the component. It's using the theme palette when that makes sense.
+     */
+    labelText: PropTypes.string,
+    /**
+     * The src of the component. It's using the theme palette when that makes sense.
      */
     src: PropTypes.string,
     /**
-     * If `true`, the button will use outline styling.
+     * If `true`, the Avatar will use outline styling.
      */
     outline: PropTypes.bool,
+    /**
+     * If `true`, the Avatar will use outline styling.
+     */
+    alt: PropTypes.string,
 };
 
 const defaultProps = {
     color: 'primary',
-    outline: false
+    outline: false,
+    className: 'avatar',
```

Comment:
If you pass `avatar` as default className, there will be 2 `avatar` class names append to the element right?